docs/manifest.json

```diff
@@ -2211,6 +2211,12 @@
               "tag": "Community",
               "href": "https://github.com/dimkl/clerk-koa/blob/main/README.md",
               "icon": "koa"
+            },
+            {
+              "title": "Angular",
+              "tag": "Community",
+              "href": "https://github.com/anagstef/ngx-clerk?tab=readme-ov-file#ngx-clerk",
+              "icon": "angular"
```

Comment:
Please can you add `angular` to the `icon` enum in `manifest.schema.json`? https://github.com/clerk/clerk-docs/blob/a67e9630e0e995755578fbf29a12584b805ad883/docs/manifest.schema.json#L91

Reply:
Will do!